che-incubator/che-code
# feat: Add 'Restart From Local Devfile' action

### What does this PR do?
Add 'Restart From Local Devfile' action

![Screenshot 2023-03-06 at 13 06 47](https://user-images.githubusercontent.com/5676062/223093826-91c5d745-9f8f-48a3-b727-c8f6517a4899.png)


### What issues does this PR fix?

https://github.com/eclipse/che/issues/18670

### How to test this PR?
- provide some changes to a devfile 
- use `Restart From Local devfile` action of the `Eclipse Che` status bar item to restart a workspace
- check that the changes are applied when the workspace is started
